src/diff/defaultRules.test.ts

```diff
@@ -120,3 +124,75 @@ describe("Version change", () => {
     verifyRule(changes[0], rule);
   });
 });
+
+describe("Examples change", () => {
+  it("does not apply ignore examples rule on a change that has no apiContract:Example change type", async () => {
```

Comment:
apiContract:Example is a node type not a change type

Reply:
Done